doc/Development_Documentation/03_Documents/01_Editables/06_Block.md

```diff
@@ -118,6 +118,23 @@ If you want to wrap buttons in a div or change the Position.
 {% do block.end() %}
 ```
 
+### Using Manual Mode with additional css class for element in editmode
+
+```twig
+{% set block = pimcore_block("gridblock", {"manual": true}).start() %}
+<div>
+    {% for b in block.iterator %}
+        {% do block.blockConstruct() %}
+            {% do block.blockStart(true, false, 'my-additional-class') %}
```

Comment:
It should be possible to skip the default parameters and [write it like this](https://twig.symfony.com/doc/3.x/templates.html#named-arguments), no?

```suggestion
            {% do block.blockStart(additionalClass='my-additional-class') %}
```

Reply:
An error is thrown when we try to skip the default parameters as mentioned. So, I request to revert the commit with the suggested change please.